sormas-backend/src/test/java/de/symeda/sormas/backend/person/PersonFacadeEjbTest.java

```diff
@@ -632,6 +634,9 @@ public void testGetFollowUpEndDatesCasesOnly() {
 		getCaseFacade().save(case12);
 		getCaseFacade().save(case2);
 
+		FeatureConfigurationIndexDto featureConfiguration =
```

Comment:
please introduce a method for activating features in `AbstractBeanTest`

and use it in all the tests that need feature activation

Reply:
added a new method to AbstractBeanTest and replaced all feature creation code from 5 test clases